internal/service/privatelinkendpointserverless/resource_privatelink_endpoint_serverless.go

```diff
@@ -79,11 +80,7 @@ func resourceCreate(ctx context.Context, d *schema.ResourceData, meta any) diag.
 	projectID := d.Get("project_id").(string)
 	instanceName := d.Get("instance_name").(string)
 
-	serverlessTenantCreateRequest := &admin.ServerlessTenantCreateRequest{
-		Comment: conversion.StringPtr("create"),
-	}
-
-	endPoint, _, err := connV2.ServerlessPrivateEndpointsApi.CreateServerlessPrivateEndpoint(ctx, projectID, instanceName, serverlessTenantCreateRequest).Execute()
+	endPoint, _, err := connV2.ServerlessPrivateEndpointsApi.CreateServerlessPrivateEndpoint(ctx, projectID, instanceName, &admin.ServerlessTenantCreateRequest{}).Execute()
```

Comment:
just curious, can you try if it works passing nil instead of  &admin.ServerlessTenantCreateRequest{} ?

Reply:
yeah I tried, that didn't work